src/js/controller/controller.js

```diff
@@ -336,13 +338,37 @@ Object.assign(Controller.prototype, {
         function _checkPlayOnViewable(model, viewable) {
             if (model.get('playOnViewable')) {
                 if (viewable) {
-                    _autoStart();
-                } else if (OS.mobile || (model.get('autoPause') && model.get('autoPause').viewability)) {
+                    _autoStart('viewable');
+                } else if (OS.mobile) {
                     _this.pause({ reason: 'autostart' });
                 }
             }
         }
 
+        function _checkPauseOnViewable(model, viewable) {
+            const instream = _this._instreamAdapter;
+
+            // If player left view and came back before ad completes,
+            // the player should begin/resume playing content.
+            if (viewable && instream && instream.noResume) {
+                instream.noResume = false;
+                model.set('playOnViewable', true);
+                return;
+            } else if (viewable || !model.get('autoPause').viewability) {
```

Comment:
`_checkPauseOnViewable` should never be called when `model.get('autoPause').viewability` is `false`.

Reply:
👍